.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -53,8 +103,10 @@ jobs:
       - name: Deploy Standalone Plugin - ${{ matrix.slug }}
         uses: 10up/action-wordpress-plugin-deploy@stable
         env:
-          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
-          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          #SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          #SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          SVN_PASSWORD: SVN_PASSWORD
+          SVN_USERNAME: SVN_USERNAME
```

Comment:
Maybe add a `TODO` here just to change this back prior to merge.

Reply:
Updated in https://github.com/WordPress/performance/pull/710/commits/6a6af58297d146aaa4d4fcfabb4ca6f1e1807713